schemas/external/repo/asset.schema.json

```diff
@@ -38,6 +45,17 @@
           "meta:usereditable": false,
           "description": "The version ID of the piece of content. It will be generated both on explicit and implicit save or upload."
         },
+        "repo:createdBy": {
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "ID of the user who initiated the action that caused the resource to be created in the repository."
```

Comment:
It's not always a user.

Reply:
Resolving without change per @lrosenthol